app/views/shared/_formbundle.html.erb

```diff
@@ -2,3 +2,7 @@
 <%= f.input :name %>
 <%= f.input :gender, as: :radio_buttons, %>
 <%= f.input :age_group, collection: ['Newborn','0 to 3 months','3 to 6 months','6 to 9 months','9 to 12 months','12 to 18 months','18 to 24 months','2T','3T'] %>
```

Comment:
Since we added @age_groups in the Bundle controller you should just be able to do @bundle.age_groups